.vscode/settings.json

```diff
@@ -57,7 +57,7 @@
   "C_Cpp.errorSquiggles": "enabled",
   "[cpp]": {
     "editor.tabSize": 4,
-    "editor.defaultFormatter": "xaver.clang-format",
+    "editor.defaultFormatter": "llvm-vs-code-extensions.vscode-clangd",
```

Comment:
```suggestion
    "editor.defaultFormatter": "xaver.clang-format",```

Reply:
Couldnt get the xaver one to work, but didn't meant to include this in git. Reverted